README.md

```diff
@@ -164,6 +164,23 @@ data = { "instances": [ model.input_example ] }
 predictions = deployment.predict(data)
 ```
 
+## Usage
+
+Usage data is collected for improving quality of the library.
+It is turned on by default if the backend is "c.app.hopsworks.ai".
```

Comment:
Call it by its name and hide the host name as a link to app.

Reply:
I am not sure if I understand your comment. Do you want me to replace `"c.app.hopsworks.ai"` with `hopsworks.constants.HOSTS.APP_HOST`?